### ISSUE TITLE
Missing CONTRIBUTION.md

### ISSUE THREAD

**Issue by hoffiz** [JamesMc86/LabVIEW-CLI#48]
Hi @JamesMc86 if I wanted to contribute to this project it will be useful to know, how you setup your development environment. I tried working from a arbitrary location, git clone <somewhere> and then tried opening the DCAF build-system common steps. Since it relies on a lot of JKI packages, it is easier to download from VIPM. If I then try to clone in "\Wiresmith Technology\LabVIEW CLI" you can see the name of the folder is different, LabVIEW-CLI vs LabVIEW CLI and the structure inside is not a match with the repo. I'm able to have both, the git source and the installed version, but need to be really careful when linking the VIs. 

I'm not suggesting it needs to change be but I'm curious how you configure you development environment. I can make changes to the installed location and the do a compare and manually pull them to the repo but it sounds strange. 

It will also be useful to know LabVIEW version so I don't recompile it too far :)

**Comment by JamesMc86** [comment]
Hi Hoffiz,

Sure that makes sense. I keep meaning to work on a template contribution guide for my LabVIEW projects anyway so I will get to it ASAP.

There will be a problem trying to use the cloned version for real use and you should use the VIPM. The reason is the VIPM will install to vi.lib which is where all dependent packages will look for it. The structures are different since this repository also contains source for the C# application.

LV version is 2011.

**Comment by hoffiz** [comment]
Thanks @JamesMc86. I appreciate the reply and no rush.

I need to clean the [documentation ](https://github.com/hoffiz/LabVIEW-CI )a bit but I think I come up with a pretty clean setup for contributing to the 2 major players needed for LabVIEW continuous integration.  LabVIEW-CLI C#/VIs and the DCAF common steps.  

VBOX machine, using VS 2017 Community and LV 2017 I'm able to rebuild your exe, I added the bin/debug to the path and tested a few dummy modifications. The only drawback is that I pushed the source to 2017 although I can revert it, for now I will leave the source in 2017.
